docs/metamacros.md

```diff
@@ -0,0 +1,66 @@
+### Docile Metamacros
+
+Docile uses an extensible metamacro syntax within docstrings to support greater flexibility.
+
+#### The syntax constist of four required parts:
+
+* `!!` double qestionmark
+* followed by `more than 0 chars` which must be a valid julia identifier
+* `(` opening bracket - parentheses
+* `)` closing bracket - parentheses
+
```

Comment:
Perhaps start with an example of the full syntax and then describe each element:

```md
Metamacros use the syntax ``!!<NAME>(<TEXT>)``.

* A `!!` prefix begins a metamacro.
* `<NAME>` must be a valid Julia identifier.
* `<TEXT>` can contain arbitrary text and is interpreted as a string.
...
```